.github/workflows-src/partials/build.yml

```diff
@@ -1,20 +1,20 @@
 - uses: actions/setup-node@v1
   with:
-    node-version: '16'
+    node-version: "16"
 - uses: actions/cache@v2
   id: yarn-cache
-  name: Load  v4 npm deps from cache
+  name: Load npm deps from cache
   with:
-    path: '**/node_modules'
+    path: "**/node_modules"
     key: ${{ runner.os }}-yarn-14-${{ hashFiles('yarn.lock') }}
 - run: yarn install --frozen-lockfile
   if: steps.yarn-cache.outputs.cache-hit != 'true'
-# v4 build
+# v5 build
 - uses: actions/cache@v2
-  id: v4-cache
+  id: site-cache
   name: Load webpack cache
   with:
-    path: 'packages/v4/.cache'
-    key: ${{ runner.os }}-v4-${{ hashFiles('yarn.lock') }}
-- run: yarn build:v4
+    path: "packages/documentation-site/.cache"
```

Comment:
I'm wondering if "site" is the best word to describe this. Is it more-so a demo (but also a site for that demo)?

Reply:
The folder is where the PF content for the site lives. I'm open to suggestions on the name. Maybe "documentation-content"? @evwilkin Thoughts?